cvat/apps/engine/migrations/0026_merge_20200417_1635.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.2.10 on 2020-04-17 16:35
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('engine', '0024_auto_20200225_0657_squashed_0029_auto_20200309_0733'),
+        ('engine', '0025_auto_20200324_1222'),
+    ]
+
+    operations = [
```

Comment:
Seems this empty migration file is not needed, could you delete it?

Reply:
Although it is an empty migration file, deleting it is resulting in migration conflicts. In that how shall I proceed?

Reply:
@Priya4607 Hi, please do the following:
1. Remove `0026_merge_20200417_1635.py`
2. Rename `0024_auto_20200225_0657_squashed_0029_auto_20200309_0733.py` -> `0026_merge_20200417_1635.py`
3. Edit `0026_merge_20200417_1635.py` file:
   - delete `replaces` string
   - replace `0023_auto_20200113_1323` with `0025_auto_20200324_1222` in the `dependencies` var

Reply:
Okay, I have made changes as suggested.